qctoolkit/serialization.py

```diff
@@ -91,6 +92,14 @@ def delete(self, identifier: str) -> None:
     def __delitem__(self, identifier: str) -> None:
         self.delete(identifier)
 
+    @abstractmethod
+    def list_contents(self) -> Set[str]:
```

Comment:
I suggest naming the method `keys`

Reply:
I think list_contents() is a more fitting name for what is going on here since the method returns a list of the (available) contents of the storage backend. keys() is a much more generic name and restricts to a plain dictionary "view" of the interface without regarding the context.. I guess it's more "pythonic" though.. ultimately, I don't really care

Reply:
I think sticking to `list_contents` is better as we are not returning a view. The return value of a keys method should update if the content of the mapping changes.